frontend/app/src/app/profile/profile.component.ts

```diff
@@ -9,10 +9,27 @@ import { USER } from '../_mock_models/mock_user'
   styleUrls: ['./profile.component.css']
 })
 export class ProfileComponent implements OnInit {
-  user = USER;
+  user : User;
+
   constructor( private accountService : AccountService) {
+    //this.accountService.user.subscribe(x => this.user = x);
+    this.user = accountService.getUser();
   }
 
   ngOnInit(): void {
   }
+
+  edit() {
+    if(document.getElementById("username").contentEditable ==  "true"){
+      document.getElementById("username").contentEditable = "false";
```

Comment:
Also here: isn't there a better way to access components parts in Angular? Genuine question.

Reply:
There is. My mind still has js set up. I will rewrite the code. I knew I forgot something. :^)